lib/fika/compiler/parser/types.ex

```diff
@@ -90,6 +90,13 @@ defmodule Fika.Compiler.Parser.Types do
     |> label("effect type")
     |> Helper.to_ast(:effect_type)
 
+  loop_type =
```

Comment:
There are still tests that need to be written for this. I mostly added it because I wanted to see if the module_compiler_test.exs suite would pass with the `get_type` refactor